AmazonChimeSDK/AmazonChimeSDKTests/audiovideo/data/transcript/TranscriptResultTests.swift

```diff
@@ -14,14 +14,17 @@ class TranscriptResultTests: XCTestCase {
     let channelId = "channel-id"
     let startTimeMs: Int64 = 1632087029249
     let endTimeMs: Int64 = 1632087029250
+    let languageCode = "en-US"
 
     func testTranscriptResultShouldInitialize() {
```

Comment:
If we are keeping these tests, wondering the motivation for having tests for result, alternative, and item but not the newly defined types (entity and languageWithScore)

Reply:
the new items were tested as part of the calls in alternative amd result, but will update to have individual tests as well for these.

Reply:
I would actually vote against having tests that purely test a constructor -- unless others disagree

Reply:
Yeah, some unit tests do not make much sense. I think we need more consideration on how to add unit tests as a separate PR if not delay RC generation today. You will need create a task to track it.